tools/tertiary-analysis/seurat/seurat_dim_plot.xml

```diff
@@ -35,6 +35,27 @@
         #if $plot_title
         --plot-title '$plot_title'
         #end if
+        #if $do_bare
+        --do-bare '$do_bare'
```

Comment:
For `boolean` param with specified `truevalue` and `falsevalue`, and if the script expects so, you can just use `--do-bare $do_bare` without putting it in a `#if ... #endif`. I am not sure but `#if $do_bare` might work in the intended way with `TRUE` and `FALSE`, as the defaults value for `boolean` param are `true` `false`.

Reply:
Great- thanks for the useful feedback @nh3 , will make some tweaks.